File: high-school/fift-year/first-semester/tpsit/Tempo/src/tempo/Linea.java
package tempo;
/**
 ***************************************************
 * LINEA
 * 
 * @author Adnan Gazi Latif
 * @brief gestisce le linee di ogni digit.
 * @date 11/04/2017
 ***************************************************
 */
import static zuclib.GraficaSemplice.*;

public class Linea {

    private Punto p1;
    private Punto p2;

    public Linea(Punto p1, Punto p2) {
        this.p1 = p1;
        this.p2 = p2;
    }

    public void disegnaLinea() {
        linea(p1.getX(), p1.getY(), p2.getX(), p2.getY(), BIANCO, 0.01);
    }
}

File: high-school/fift-year/first-semester/informatica/ReteTrasporti/public_html/Linea.js
/********************************************************
 *LINEA
 *@author Gazi Adnan Latif, Riccardo Fusaro
 *@brief Contiene informazioni di ogni linea di autobus.
 *@date 12/05/2018
********************************************************/
function Linea (n, f, c){
    this.name = n;
    this.fermate = f;
    this.colore = c;
}

Linea.prototype.getName=function(){
    return this.name;
}

Linea.prototype.getColore=function(){
    return this.colore;
}

Linea.prototype.setColore=function(newColore){
    this.colore = newColore;
}

Linea.prototype.getFermate=function(pos){
    return this.fermate[pos];
}

Linea.prototype.setName=function(newName){
    this.name = newName;
}

Linea.prototype.setFermate=function(newFermata, pos){
    this.fermate[pos] = newFermata;
}